### ISSUE TITLE
Output list of AidStream organisation accounts to decide which need migrating

### ISSUE THREAD

**Issue by emmajclegg** [younginnovations/iatipublisher#1533]
This task follows [#1479](https://github.com/younginnovations/iatipublisher/issues/1479) (analysing the AidStream user base) and will inform #1489 (splitting accounts into batches for migration).

Can we export a list of AidStream organisation accounts with the following information:
- Organisation name
- Organisation identifier
- Account type (free vs. pro)
- Last login date (by any user)
- Number of users
- Is the account linked to an IATI Registry publisher account? (yes/no)
- Do they have an organisation file published?
- Number of published activities
- Number of draft (not published) activities
- Number of (activity) transactions 
- Do they have activity results data? (yes/no)

This list should exclude:
- Disabled accounts
- Test accounts
- Accounts that have already been migrated to IATI Publisher

How will this list be helpful?:
- so that I can start entering numbers of accounts in the [Account Migration plan](https://docs.google.com/spreadsheets/d/17GsCBOXDgmYS56QGFAqAn7gbXjkXUEHFwuuIxW0yPnM/edit?gid=442310768#gid=442310768)
- To give us a more balanced view of how much activity data an account will have to migrate (beyond just looking at activity volumes)
- To identify and exclude accounts that do not need migrating

We will likely want to rerun this analysis closer to migration dates, to make sure any new accounts are picked up, and that the information is up to date. Therefore, making it scripted (i.e. easy to rerun) is beneficial.

**Comment by BibhaT** [comment]
Hi Emma, Sanil has extracted a file with all the information mentioned above. I've shared the file with you and the team. Here's the link for the same: https://docs.google.com/spreadsheets/d/1p3Nz58Me77w2MCeb0IrLoXu46J5Zj49uZKrsb7SNe9g/edit?usp=sharing

**Comment by emmajclegg** [comment]
Thanks Bibha. I've used this to update numbers in the provisional [account migration plan](https://docs.google.com/spreadsheets/d/17GsCBOXDgmYS56QGFAqAn7gbXjkXUEHFwuuIxW0yPnM/edit?gid=442310768#gid=442310768).

I'm not sure whether we should group accounts by total activities (i.e. draft + published), or just published activities, but have left it as the latter for now. I've also separated out accounts with a single published activity, as that forms a large group in itself.
